src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -73,47 +72,48 @@ class CloseAccountPage extends Component {
                         styles.p5,
                     ]}
                 >
-                    <Text>{this.props.translate('closeAccountPage.reasonForLeavingPrompt')}</Text>
-                    <TextInput
-                        multiline
-                        numberOfLines={6}
-                        textAlignVertical="top"
-                        value={this.state.reasonForLeaving}
-                        onChangeText={reasonForLeaving => this.setState({reasonForLeaving})}
-                        label={this.props.translate('closeAccountPage.enterMessageHere')}
-                        containerStyles={[styles.mt5, styles.closeAccountMessageInput]}
-                    />
-                    <Text style={[styles.mt5]}>
-                        <Text style={[styles.textStrong]}>
-                            {this.props.translate('closeAccountPage.closeAccountWarning')}
+                    <View style={[styles.flexGrow1]}>
```

Comment:
I am not sure why we need this `View`, when we have a `ScrollView` with `flexGrow1`

Reply:
The `Button` is in `ScrollView`, we need the `View` here.
Basicly:
Red: `ScrollView`
Green: `View`
Orange: `Button`

<img width="618" alt="Screen Shot 2022-09-19 at 9 36 09 AM" src="https://user-images.githubusercontent.com/11542415/190950684-5e14af47-1adb-41fc-a238-6d2041eebb04.png">

Reply:
It will look like this if we remove the view
<img width="618" alt="Screen Shot 2022-09-19 at 9 39 25 AM" src="https://user-images.githubusercontent.com/11542415/190950776-01111595-a844-43bf-8f9a-7289efa7937f.png">

Reply:
Before you change, lets check with @shawnborton. Shawn can you confirm the expected behavior on the position of the button.

I am guessing it should be first one inline with other screens.

Reply:
We want the button to appear bottom-docked to the screen.

Reply:
Thanks @shawnborton. @kakajann then we won't need any change here.

Reply:
Alright